iframe/index.html

```diff
@@ -6,45 +6,81 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Authsio</title>
-  <link rel="stylesheet" href="style.css">
-  <link rel="stylesheet" href="custom.css">
+  <script src="https://cdn.tailwindcss.com?plugins=forms,typography,aspect-ratio,line-clamp"></script>
```

Comment:
Installing via CDN as we are currently prototyping. When validated, Tailwind CSS will be setup in the build system.

Reply:
Great idea.